src/charts/Chart.ts

```diff
@@ -257,15 +272,18 @@ abstract class Chart {
             propertyZ = this.config.get('propertyZ'),
             propertyKey = this.config.get('propertyKey'),
             propertyStart = this.config.get('propertyStart'),
-            propertyEnd = this.config.get('propertyEnd');
+            propertyEnd = this.config.get('propertyEnd'),
+            propertyError = this.config.get('propertyError'),
```

Comment:
The error should be an 'event' instead of a property, since it's optional and may not be always present